include/sway/config.h

```diff
@@ -206,7 +206,7 @@ struct bar_config {
 struct border_colors {
 	float border[4];
 	float background[4];
-	float text[4];
+	uint32_t text; /**< uint32_t because that's the format that cairo uses */
```

Comment:
This isn't the appropriate time for a `/**<` comment

Reply:
Does `/**<` have any special meaning over the standard `//`? I copied the comment format from a few lines below because I thought there's no special meaning and it's just a style thing.

Reply:
We informally use it for documentation under the assumption that maybe someone will export them with some kind of generator